test/workbox-cli/node/app.js

```diff
@@ -10,7 +10,7 @@ describe(`[workbox-cli] app.js`, function() {
   const PROXIED_CONFIG_FILE = path.resolve(process.cwd(), '/will/be/proxied');
   const PROXIED_ERROR = new Error('proxied error message');
   const PROXIED_CONFIG = {};
-  const INVALID_CONFIG_FILE = '/does/not/exist';
+  const INVALID_CONFIG_FILE = path.resolve(process.cwd(), '/does/not/exist');
```

Comment:
Will windows be ok with the forward slashes?

Reply:
`path.resolve()` converts them to whatever the OS uses, so it will be okay with this change in place.

Reply:
...at least it should, since I'm doing the same thing earlier, but this PR's AppVeyor run didn't complete due to #883